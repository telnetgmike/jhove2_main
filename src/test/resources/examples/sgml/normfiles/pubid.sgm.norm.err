/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:17:73:1844095592.338:W: cannot generate system identifier for general entity "uml"
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:17:73:1844095592.25:E: general entity "uml" not defined and no default entity
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:17:73: relevant clauses:  ISO 8879:1986 9.4.4.1p2
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:17:76:1844095592.325:E: reference to entity "uml" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:17:72: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:24:30:1844095592.338:W: cannot generate system identifier for general entity "dollar"
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:24:30:1844095592.25:E: general entity "dollar" not defined and no default entity
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:24:30: relevant clauses:  ISO 8879:1986 9.4.4.1p2
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:24:36:1844095592.325:E: reference to entity "dollar" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:24:29: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:729:1844095592.338:W: cannot generate system identifier for general entity "copy"
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:729:1844095592.25:E: general entity "copy" not defined and no default entity
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:729: relevant clauses:  ISO 8879:1986 9.4.4.1p2
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:733:1844095592.325:E: reference to entity "copy" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:728: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:799:1844095592.338:W: cannot generate system identifier for general entity "lsqb"
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:799:1844095592.25:E: general entity "lsqb" not defined and no default entity
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:799: relevant clauses:  ISO 8879:1986 9.4.4.1p2
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:803:1844095592.325:E: reference to entity "lsqb" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:798: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:826:1844095592.338:W: cannot generate system identifier for general entity "rsqb"
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:826:1844095592.25:E: general entity "rsqb" not defined and no default entity
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:826: relevant clauses:  ISO 8879:1986 9.4.4.1p2
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:830:1844095592.325:E: reference to entity "rsqb" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:825: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:80:20:1844095592.338:W: cannot generate system identifier for general entity "minus"
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:80:20:1844095592.25:E: general entity "minus" not defined and no default entity
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:80:20: relevant clauses:  ISO 8879:1986 9.4.4.1p2
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:80:25:1844095592.325:E: reference to entity "minus" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:80:19: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:82:315:1844095592.325:E: reference to entity "minus" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:80:19: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:82:328:1844095592.325:E: reference to entity "minus" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:80:19: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:82:378:1844095592.325:E: reference to entity "minus" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:80:19: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:250:1844095592.338:W: cannot generate system identifier for general entity "ohgr"
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:250:1844095592.25:E: general entity "ohgr" not defined and no default entity
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:250: relevant clauses:  ISO 8879:1986 9.4.4.1p2
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:254:1844095592.325:E: reference to entity "ohgr" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:249: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:501:1844095592.338:W: cannot generate system identifier for general entity "prop"
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:501:1844095592.25:E: general entity "prop" not defined and no default entity
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:501: relevant clauses:  ISO 8879:1986 9.4.4.1p2
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:505:1844095592.325:E: reference to entity "prop" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:500: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:511:1844095592.325:E: reference to entity "ohgr" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:249: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:584:1844095592.325:E: reference to entity "prop" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:500: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:593:1844095592.325:E: reference to entity "ohgr" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:249: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:619:1844095592.338:W: cannot generate system identifier for general entity "tgr"
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:619:1844095592.25:E: general entity "tgr" not defined and no default entity
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:619: relevant clauses:  ISO 8879:1986 9.4.4.1p2
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:622:1844095592.325:E: reference to entity "tgr" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:618: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:704:1844095592.325:E: reference to entity "minus" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:80:19: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:717:1844095592.325:E: reference to entity "prop" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:500: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:727:1844095592.325:E: reference to entity "ohgr" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:249: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:775:1844095592.325:E: reference to entity "tgr" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:618: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:84:1085:1844095592.325:E: reference to entity "lsqb" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:798: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:84:1126:1844095592.325:E: reference to entity "rsqb" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:825: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:90:580:1844095592.338:W: cannot generate system identifier for general entity "deg"
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:90:580:1844095592.25:E: general entity "deg" not defined and no default entity
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:90:580: relevant clauses:  ISO 8879:1986 9.4.4.1p2
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:90:583:1844095592.325:E: reference to entity "deg" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:90:579: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:91:887:1844095592.338:W: cannot generate system identifier for general entity "plusmn"
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:91:887:1844095592.25:E: general entity "plusmn" not defined and no default entity
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:91:887: relevant clauses:  ISO 8879:1986 9.4.4.1p2
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:91:893:1844095592.325:E: reference to entity "plusmn" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:91:886: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:92:219:1844095592.325:E: reference to entity "deg" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:90:579: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:92:409:1844095592.325:E: reference to entity "deg" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:90:579: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:92:641:1844095592.325:E: reference to entity "minus" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:80:19: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:98:427:1844095592.325:E: reference to entity "ohgr" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:249: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:99:320:1844095592.325:E: reference to entity "ohgr" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:83:249: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:100:735:1844095592.325:E: reference to entity "lsqb" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:798: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:100:823:1844095592.325:E: reference to entity "rsqb" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:825: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:100:1195:1844095592.325:E: reference to entity "lsqb" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:798: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:100:1257:1844095592.325:E: reference to entity "rsqb" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:63:825: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:106:639:1844095592.325:E: reference to entity "deg" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:90:579: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:117:68:1844095592.338:W: cannot generate system identifier for general entity "ngr"
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:117:68:1844095592.25:E: general entity "ngr" not defined and no default entity
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:117:68: relevant clauses:  ISO 8879:1986 9.4.4.1p2
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:117:71:1844095592.325:E: reference to entity "ngr" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:117:67: entity was defined here
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:233:348:1844095592.325:E: reference to entity "deg" for which no system identifier could be generated
/usr/bin/sgmlnorm:/cygdrive/c/jhove2_hg_smm/src/test/resources/examples/sgml/sgmlfiles/somefile.sgm:90:579: entity was defined here
