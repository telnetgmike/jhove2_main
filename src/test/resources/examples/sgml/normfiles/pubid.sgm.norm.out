<!DOCTYPE ARTICLE PUBLIC "-//ISO 99999:9999//DTD dtd0300//EN">
<ARTICLE>
<FRONT>
<TITLEGRP>
<TITLE>:LORED ipsmum bonum</TITLE>
</TITLEGRP>
<AUTHGRP>
<AUTHOR>
<FNAME>John </FNAME>
<SURNAME>Homer</SURNAME>
<PUNCT> and </PUNCT>
</AUTHOR>
<AUTHOR>
<FNAME>John Q. </FNAME>
<SURNAME>Public </SURNAME>
</AUTHOR>
<AFFGRP>
<AFF>
<ORGNAME>Chemical stuff</ORGNAME>
</AFF>
</AFFGRP>
</AUTHGRP>
<AUTHGRP>
<AUTHOR>
<FNAME>JANE Q. </FNAME>
<SURNAME>PRIVATE </SURNAME>
</AUTHOR>
<AFFGRP>
<AFF>
<ORGNAME>MYLORG, D-76287&thinsp;Rheinstetten, Germany </ORGNAME>
</AFF>
</AFFGRP>
</AUTHGRP>
<PUBFRONT>
<TCLINK>obufs</TCLINK>
<ULINK>9999</ULINK>
<ARTID>9999p</ARTID>
<STITLE>J.Of. Someth.</STITLE>
<CCC>999-99999/0000000</CCC>
<PACYR></PACYR>
<PAC></PAC>
<CODEN>DONON6</CODEN>
<VOLUME>1</VOLUME>
<ISSUE>1</ISSUE>
<FPAGE>1</FPAGE>
<LPAGE>1</LPAGE>
<SISAC>eomwetsuf1L.1;1</SISAC>
<ISSN>0000-0000</ISSN>
<SEQ>1</SEQ>
<IDATE>1 January 1000</IDATE>
<CPYRT>
<DATE>1000</DATE>
<CPYRTNME>
<ORGNAME>American Institute of Something or other/ORGNAME>
</CPYRTNME>
</CPYRT>
<HISTORY>
<RECEIVED>
<DATE>01 Jan 1000</DATE>
</RECEIVED>
<ACCEPTED>
<DATE>01 Jan 1000</DATE>
</ACCEPTED>
</HISTORY>
<DOI>10.9999/9.999999</DOI>
<AIPKEY>1.49999999999999999/AIPKEY>
</PUBFRONT>
<SECTHEAD>
<HD1>COMMUNICATIONS</HD1>
</SECTHEAD>
<SPIN>
<DOCANAL TYPE="ME" KIND="KL">
<PACYR>1000</PACYR>
<PAC>3325+k</PAC>
</DOCANAL>
<JOUIDX>
<COUNTRY>UT</COUNTRY>
<TOTALIDX>2</TOTALIDX>
<ADDIDX>magnetic </ADDIDX>
<ADDIDX>effects</ADDIDX>
</JOUIDX>
</SPIN>
<ABSTRACT>
<P>For stuff <FORMULA><SUP><ROMAN>13</ROMAN></SUP><ROMAN>C</ROMAN></FORMULA> studies.  <EMPH TYPE="1">1 Institute of something.</EMPH>  <II>somesuff</II></P>
</ABSTRACT>
</FRONT>
<BODY>
<PART>
<CHAPTER>
<TITLE>INTRODUCTION</TITLE>
<P> elaxation time, <FORMULA>T<INF>1</INF>,</FORMULA>  se of dipolar solids the values of <FORMULA>T<INF>1</INF></FORMULA> can be so long that initial l hours.<CITEREF RID="R1" STYLE="SUPERIOR">1</CITEREF> Even  relatively short values of <FORMULA>T<INF>1</INF></FORMULA> of tructural determinations spectroscopy.<CITEREF RID="R2" STYLE="SUPERIOR">2</CITEREF><CITEREF RID="R3" STYLE="SUPERIOR">3</CITEREF> eral such methods have been reported.  to liquids.<CITEREF RID="R4" STYLE="SUPERIOR">4</CITEREF> A less well known, and certainly more complicated samples.<CITEREF RID="R5" STYLE="SUPERIOR">5</CITEREF><CITEREF RID="R6" STYLE="SUPERIOR">6</CITEREF><CITEREF RID="R7" STYLE="SUPERIOR">7</CITEREF> Ultrasound has also been used to reduce <FORMULA>T<INF>1</INF></FORMULA> in solids.<CITEREF RID="R8" STYLE="SUPERIOR">8</CITEREF> The present paper proposes a method of reducing <FORMULA>T<INF>1</INF></FORMULA> for dipolar nuclei that, although relying on the well-known dependence of <FORMULA>T<INF>1</INF></FORMULA> on the previously. </P>
</CHAPTER>
</PART>
</BODY>
<BACK>
<BIBLIST>
<CITATION ID="R1"><REF>
<ART>
<OTHINFO>See, e.g.</OTHINFO>
<PUNCT>, </PUNCT>
<AUTHGRP>
<AUTHOR>
<FNAME>D. C. </FNAME>
<SURNAME>Public</SURNAME>
</AUTHOR>
<PUNCT>, and </PUNCT>
<AUTHOR>
<FNAME>D. P. </FNAME>
<SURNAME>Private</SURNAME>
</AUTHOR>
</AUTHGRP>
<PUNCT>, </PUNCT>
<JOURNAL>J. Am. Something. Soc. </JOURNAL>
<CODEN>JJJJJJ/CODEN>
<VOL>1</VOL>
<PUNCT>, </PUNCT>
<PP>1</PP>
<PUNCT> (</PUNCT>
<YEAR>1000</YEAR>
<PUNCT>).</PUNCT>
</ART>
</REF></CITATION>
</BIBLIST>
</BACK>
</ARTICLE>
